api/src/api/admin.js

```diff
@@ -97,13 +98,29 @@ router.delete(
 router.post(
   '/resources',
   errorWrap(async (req, res) => {
-    if (req.body.image && req.body.image.length > 0) {
-      const link = await imageHelper(req.body.image);
+    const { image, address, city, state, zip } = req.body;
+
+    if (image && image.length > 0) {
+      const link = await imageHelper(image);
       if (link) {
         req.body.image = link;
       }
     }
 
+    if (
+      (address && address.length > 0) ||
+      (city && city.length > 0) ||
+      (state && state.length > 0) ||
+      (zip && zip.length > 0)
+    ) {
+      const [long, lat] = await extractLongLat(
+        `${address},${city},${state},${zip}`,
+      );
+      if (long != null && lat != null) {
```

Comment:
Something I wanted to ask – are we just checking for nullish values (`null` and `undefined`) here? Or only `null`. If nullish values, we can simplify to 

```js
if (long && lat)
```

otherwise we need to explicitly check only `null` with 


```js
if (long !== null && lat !== null)
```

This goes for rest of repo too :)